src/token/raft.ts

```diff
@@ -66,7 +79,7 @@ export class RaftToken {
    * @param period The period, in year.
    * @param options.veRaftTotalSupply The total supply of veRAFT. If not provided, will query.
    * @param options.annualGiveAway The annual give away of RAFT. If not provided, will query.
-   * @returns The collateral ratio. If the debt is 0, returns the maximum decimal value (represents infinity).
+   * @returns The estimated staked APR.
    */
   public async estimateStakeApr(
     stakeAmount: Decimal,
```

Comment:
```suggestion
   * @returns The estimated staking APR.
   */
  public async estimateStakingApr(
    stakingAmount: Decimal,
```

Reply:
this will impact the app side. if we want to rename this i can do this in next PR

Reply:
Please rename